### ISSUE TITLE
Logits output from teacher

### ISSUE THREAD

**Issue by cpsu00** [castorini/distill-bert#3]
Hello, I would like to ask if you can share your logits output file from the teacher model. Thank you!
